### ISSUE TITLE
Add unlink

### ISSUE THREAD

**Issue by michaelplemmons** [jlevy/the-art-of-command-line#432]
From the rm/unlink man page.

     When the utility is called as unlink, only one argument, which must not be a direc-
     tory, may be supplied.  No options may be supplied in this simple mode of operation,
     which performs an unlink(2) operation on the passed argument.

This can be a safe way to remove symlinks to directories without following the directory.

**Comment by jlevy** [comment]
Agreed. I think for this to be handled right, we need one or two bullet points about removing files and symlinks, including this gotcha. It should cover

- Take care when using `rm -r` when symlinks are involved. To remove a symlink only (especially if the target is a directory), use `unlink`.
- Use `rmdir` as a safe way to make sure the directory you're deleting is empty.

More:
http://stackoverflow.com/questions/210120/remove-a-symlink-to-a-directory

Comments/PRs welcome.
